.travis.yml

```diff
@@ -17,9 +16,9 @@ matrix:
     allow_failures:
         - php: 7.0
     include:
-        - php: 5.3
+        - php: 5.4
```

Comment:
Rebase with master, `.travis.yml` changed a lot after my last changes. Also we're bumping straight to PHP 5.5 and Symfony 2.7.